modules/imgproc/src/hough.cpp

```diff
@@ -882,6 +881,700 @@ void cv::HoughLinesP(InputArray _image, OutputArray _lines,
     Mat(lines).copyTo(_lines);
 }
 
+/****************************************************************************************\
+*                                     Circle Detection                                   *
+\****************************************************************************************/
+template<> class DataType<Point>
```

Comment:
What is the problem?
This helper is already declared [here](https://github.com/opencv/opencv/blob/3.3.1/modules/core/include/opencv2/core/types.hpp#L194) (deprecated, will be removed).
New type traits is [here](https://github.com/opencv/opencv/blob/3.3.1/modules/core/include/opencv2/core/types.hpp#L214-L217).